include/ajax.tasks.php

```diff
@@ -113,7 +113,7 @@ function add($tid=0, $vars=array()) {
                 if ($desc
                         && $desc->isAttachmentsEnabled()
                         && ($attachments=$desc->getWidget()->getAttachments()))
-                    $vars['files'] = $attachments->getFiles();
+                $vars['files'] = $attachments->getFiles();
```

Comment:
Looks like whitespace issue here but want to point out that the line should be tapped in since it's part of `if` statement.